server.js

```diff
@@ -1,52 +1,26 @@
 'use strict';
 
-const pg = require('pg');
+const books = require('./books.js')

```

Comment:
you don't need the `.js` extension on this as you are requiring in a module not a file